packages/admin-next/admin-shared/package.json

```diff
@@ -5,14 +5,17 @@
   "author": "Kasper Kristensen <kasper@medusajs.com>",
```

Comment:
Maybe the timing is not the best, but I feel like we can do better with the naming. To me
`admin-sdk` would be closer to what `admin-shared` does. And the current `admin-sdk` can be more something like `admin-builder`. I am not saying we need to do it in this PR, but something to think about before officially releasing v2.

Reply:
Agree that the naming can be better, and something we need to revisit, but would hold off on it to post RC, as it requires we update the packages a couple of different places.